common/sources.gni

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_chrome_common_sources = [
```

Comment:
We should put a comment describing why some sources live in `BUILD.gn` and some are here, and which way people should prefer when adding new entries.

Reply:
I just followed the existing sources.gni files in //brave. But, yes, it's better to add a comment. Updated.